src/Logging/Logging.Testing/src/build/Microsoft.Extensions.Logging.Testing.props

```diff
@@ -9,15 +9,21 @@
   <Target Name="SetLoggingTestingAssemblyAttributes"
     BeforeTargets="GetAssemblyAttributes"
     Condition="'$(GenerateLoggingTestingAssemblyAttributes)' != 'false'">
+    <PropertyGroup>
+      <PreserveExistingLogsInOutput Condition="'$(PreserveExistingLogsInOutput)' == '' AND '$(ContinuousIntegrationBuild)' == 'true'">true</PreserveExistingLogsInOutput>
```

Comment:
The idea is that we will preserve existing logs on CI but delete existing logs locally by default. These can always be overridden if needed.